psi4/share/psi4/scripts/test_threading.py

```diff
@@ -184,7 +189,7 @@ def print_math_ldd(args):
 
     module, sharedlibrary = args.module.split('/')
     mod = importlib.import_module(module)
-    modcore = mod.__file__[:-11] + sharedlibrary
+    modcore = os.path.dirname(os.path.abspath(mod.__file__)) + os.path.sep + sharedlibrary
```

Comment:
+1, other thing to consider is `os.path.join()` instead of ` + os.path.sep + `. Its more of a stylistic thing, but I always found it cleaner.

Reply:
I should remember that one and use it instead of `os.path.sep.join` for 3+